serverless/aws/lambda-cats-effect/src/main/scala/sttp/tapir/serverless/aws/lambda/runtime/AwsLambdaRuntimeInvocation.scala

```diff
@@ -82,10 +85,11 @@ object AwsLambdaRuntimeInvocation extends StrictLogging {
     val sendResult: (RequestEvent, Either[Throwable, AwsResponse]) => F[Unit] = (event, result) =>
       result match {
         case Right(response) =>
-          logger.info(s"Request event ${event.requestId} completed successfully")
+          logger.info("Request event {} completed successfully", event.requestId)
```

Comment:
**nitpick:** While the slf4j's formatter's deferred evaluation can help to avoid evaluating expensive expressions, I don't think it should be used in all cases. Here we have a simple expression (`event.requestId`) and `INFO` level, so it will always be evaluated anyway. Using interpolated string where possible makes the message much more readable in the code IMO. (This comment applies to other usages where I don't see a clear gain, like `logger.error` calls above and below).